cluster/mutex.go

```diff
@@ -161,10 +162,23 @@ func (m *Mutex) refreshLock() error {
 // Lock locks m. If the mutex is already locked by any plugin instance, including the current one,
 // the calling goroutine blocks until the mutex can be locked.
 func (m *Mutex) Lock() {
+	m.LockWithContext(context.Background())
+}
+
+// Lock locks m unless the context is cancelled. If the mutex is already locked by any plugin
```

Comment:
Nit: function name in comment
```suggestion
// LockWithContext locks m unless the context is cancelled. If the mutex is already locked by any plugin
```